freqtrade/freqtradebot.py

```diff
@@ -1263,29 +1297,29 @@ def _notify_exit(self, trade: Trade, order_type: str, fill: bool = False) -> Non
             'gain': gain,
             'limit': profit_rate,
             'order_type': order_type,
-            'amount': trade.amount,
-            'open_rate': trade.open_rate,
-            'close_rate': trade.close_rate,
+            'amount': amount,
+            'open_rate': open_rate,
+            'close_rate': profit_rate,
             'current_rate': current_rate,
-            'profit_amount': profit_trade,
+            'profit_amount': profit,
             'profit_ratio': profit_ratio,
             'buy_tag': trade.buy_tag,
             'sell_reason': trade.sell_reason,
             'open_date': trade.open_date,
             'close_date': trade.close_date or datetime.utcnow(),
+            'stake_amount': trade.stake_amount,
             'stake_currency': self.config['stake_currency'],
             'fiat_currency': self.config.get('fiat_display_currency', None),
+            'sub_trade': sub_trade,
         }
-
-        if 'fiat_display_currency' in self.config:
-            msg.update({
-                'fiat_currency': self.config['fiat_display_currency'],
-            })
+        if sub_trade:
+            msg['cumulative_profit'] = trade.realized_profit
```

Comment:
we only provide when it is a partial sell.

Reply:
i do understand that from the code - but is there a downside to providing it all the time?
it'd make both this method, as well as methods using this simpler, as they can rely on this being available.

Reply:
> is there a downside to always provide this? 

Nope;  Need to add in all test cases 

> if it's a simple scenario (full buy followed by a full sell) - this would simply be 0 - correct?
Yes